using_listwheelscrollview/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Jun 23 08:50:38 CEST 2017
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip

```

Comment:
What is this? 

![Screenshot 2020-10-06 at 01 01 17](https://user-images.githubusercontent.com/2096087/95140555-7758ba80-076f-11eb-95ff-8a46e4acc880.png)

Reply:
I don't know.😅

Reply:
Revert changes.

Reply:
sir but these are not showing in the file
![Screenshot (3110)](https://user-images.githubusercontent.com/62784464/95213781-d28cba80-080c-11eb-955f-eb8a8e5b6c31.png)

Reply:
maybe the file is important any reveting the changes may lead to bug.